README.md

```diff
@@ -217,6 +217,19 @@ Options:
   --help  Show this message and exit.
 ```
 
+### `comments`
```

Comment:
I've been trying to keep the order here matching the output of `gogettr --help`, so this should be the second entry?